tensorboard/uploader/proto/write_service.proto

```diff
@@ -314,7 +348,7 @@ message WriteBlobRequest {
 }
 
 message WriteBlobResponse {
-  enum ObjectState {
+  enum BlobState {
```

Comment:
Do we actually need to report the BlobState in WriteBlobResponse any more?  I guess just to confirm that the `finalize_object` instruction was accepted?  IMO just getting a Status OK response to a message that contained the finalization instruction should probably be good enough, but I guess we could retain as a sanity check.

Reply:
Yeah, I figured they're a super cheap sanity check, so why not.